pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -136,7 +136,8 @@ enum ProtocolVersion {
 	v10 = 10;// Added proxy to broker
 	v11 = 11;// C++ consumers before this version are not correctly handling the checksum field
 	v12 = 12;// Added get topic's last messageId from broker
-                 // Added CommandActiveConsumerChange
+			 // Added CommandActiveConsumerChange
+			 // Added CommandGetTopicsOfNamespace
```

Comment:
`v12` got released in 1.22. We should bump the protocol version again

Reply:
Thanks. will change it.

Reply:
oh, checked again, seems 1.22 still use v11:
https://github.com/apache/incubator-pulsar/blob/branch-1.22/pulsar-common/src/main/proto/PulsarApi.proto#L137